install/kubernetes/tetragon/templates/operator_role.yaml

```
@@ -0,0 +1,28 @@
+{{- if and (and .Values.tetragonOperator.enabled .Values.serviceAccount.create) (gt (int .Values.tetragonOperator.replicas) 1) }}
```

maybe it's a bit more explicit to check if `tetragonOperator.failoverLease.Enabled` is set to true:
```suggestion
{{- if and (and .Values.tetragonOperator.enabled .Values.serviceAccount.create) .Values.tetragonOperator.failoverLease.Enabled }}
```